addons/mail/static/src/models/rtc.js

```diff
@@ -644,8 +644,11 @@ registerModel({
                     if (!rtcSession) {
                         continue;
                     }
-                    const rtcDataChannel = rtcSession.rtcDataChannel;
-                    if (!rtcDataChannel || rtcDataChannel.dataChannel.readyState !== 'open') {
+                    const rtcDataChannel = this.messaging.models['RtcDataChannel'].insert({
+                        label: "notifications",
+                        rtcPeerConnection: replace(rtcSession.rtcPeerConnection),
+                    });
```

Comment:
...But then why the `label` is forged here, and don't just use `RtcDataChannel/dataChannel/label`?